apps/divorce/div-cos/demo.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     java:
       ingressClass: traefik-private
       disableIngressClassAnnotation: true
-      image: hmctspublic.azurecr.io/div/cos:prod-dcafa8a-20221209092054 #{"$imagepolicy": "flux-system:div-cos"}
+      image: hmctspublic.azurecr.io/div/cos:pr-1727-3cebbec-20230115112410 #{"$imagepolicy": "flux-system:div-cos"}
```

Comment:
imagePolicy should be `demo-div-cos` instead of `div-cos` similarly other.  Get confirm with Damon

Reply:
Thanks Rishi! Resolved 😃